compiler/qsc_linter/src/linter/ast.rs

```diff
@@ -0,0 +1,185 @@
+use qsc_ast::{
```

Comment:
You'll need to include the standard MIT license header in these new files (see other files for reference)

Reply:
Nice catch! Fixed. I wonder if there is way to make the compilation fail if those are missing @swernli